components/dashboard/src/settings/settings.routes.ts

```diff
@@ -19,5 +19,6 @@ export const settingsPathTeamsJoin = [settingsPathTeams, "join"].join("/");
 export const settingsPathTeamsNew = [settingsPathTeams, "new"].join("/");
 
 export const settingsPathVariables = "/variables";
+export const settingsPathPersonalAccessTokens = "/personal-access-tokens";
```

Comment:
`/pat` would also be an alternative, but I kind of like the full length. We could also shorten it to `/personal-tokens`. Github uses `/tokens`

Reply:
Was hoping for this suggestion, as I like that it's explicit but it is a tad long. Let's go with `personal-tokens`